app/src/lib/git/core.ts

```diff
@@ -186,14 +187,13 @@ export async function git(
       : false
     if (!acceptableExitCode) {
       gitError = GitProcess.parseError(result.stderr)
-      if (!gitError) {
```

Comment:
I noticed this is a numeric error, but can also be null… and `BadConfigValue` was added at the beginning of the enum so its value is `0` 😅